### PR TITLE
Fixed a bug in template file hubbard.py

### PR DESCRIPTION
**Pull request by donrolih**
This is a fix of this issue: #3

### CODE REVIEW

File: doc/guide/hubbard.py [hunk 1]
```diff
@@ -72,7 +72,7 @@
 
 # Model Parameters
 mp = { "U1": U, "B1": B, "omega": omega, "g1": g1, "n1": n1 }
-for k,v in mp.items():
+for k,v in mp.copy().items():
```
Comment (Wentzell) [inline doc/guide/hubbard.py]: How about making this a single line as opposed to the loop?
```python
mp = { k: v for k,v in mp.items() if v is not None }
```

Comment (Wentzell) [reply doc/guide/hubbard.py]: @donrolih Any thought?

Comment (donrolih) [reply doc/guide/hubbard.py]: Yes, I agree and have updated the fix.

**Review by Wentzell — APPROVED** [review]
Thank you @donrolih for this fix!

**Pull request merged**